src/md-inject.ts

```diff
@@ -27,14 +28,22 @@ interface ReplaceOptions {
   followSymbolicLinks: boolean
   globPattern: string
   quiet: boolean
+  systemEnvironment: boolean
 }
 
 const main = async (
-  { blockPrefix, followSymbolicLinks, globPattern, quiet }: ReplaceOptions = {
+  {
+    blockPrefix,
+    followSymbolicLinks,
+    globPattern,
+    quiet,
+    systemEnvironment,
```

Comment:
do you think its worth re-assigning this option to `useSystemEnvironment`, as we do in `prepareEnvironment` ?